src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const rows = sourceString.split(';').filter(Boolean);
+
+  for (const row of rows) {
+    const [key, value] = row.split(':');
```

Comment:
Be cautious when using 'split' to parse the key-value pairs. If a value contains a colon ':', it will be split incorrectly. Although handling this case is not required for the task, it's something to be aware of in a real-world scenario.